src/modules/shareAlliancePost/assets/util.ts

```diff
@@ -154,14 +154,21 @@ export function createIcon(
     return iconElement;
 }
 
-export function createEditBtn(editBtnClass: string) {
+export function createEditBtn(
+    editBtnClass: string[] | string = [],
+    transform = true
+) {
     const editBtn = document.createElement('button');
-    editBtn.classList.add('btn', 'btn-xs', 'btn-default', editBtnClass);
-    editBtn.style.setProperty('position', 'absolute');
-    editBtn.style.setProperty('right', '0');
-    editBtn.style.setProperty('transform', 'translateY(-3px)');
+    editBtn.classList.add('btn', 'btn-xs', 'btn-default');
+    if (Array.isArray(editBtnClass)) editBtn.classList.add(...editBtnClass);
+    else editBtn.classList.add(editBtnClass);
+    if (transform) {
```

Comment:
Why check if its true every time?

Reply:
only default value is true but parameter can be changed